litestar/_signature/models/attrs_signature_model.py

```diff
@@ -75,6 +75,16 @@ def _pass_through_unstructure_hook(value: Any) -> Any:
     return value
 
 
+def _structure_bool(value: Any, cls: type[bool]) -> bool:
+    if isinstance(value, cls):
+        return value
+
+    if isinstance(value, str):
+        return value.lower() in ("true", "1")
+
+    return cls(value)
```

Comment:
Should this raise here @Goldziher?

Reply:
It will raise a value error? If so it's gone as is, if not than yes

Reply:
It probably should raise for arbitrary strings too.

Reply:
this is all the symbols they mention in the pydantic docs now